src/middleware/getTeamMembers.ts

```diff
@@ -0,0 +1,26 @@
+import { getPremiumStatus, getTeamMembers as getTeamMembersRequest } from '../endpoints';
+import { Secrets } from '../types';
+
+interface GetTeamMembersParams {
+    secrets: Secrets;
+    page: number;
+}
+
+export const getTeamMembers = async (params: GetTeamMembersParams) => {
+    const { secrets, page } = params;
+
+    const premiumStatus = await getPremiumStatus({
+        secrets,
+    });
+    const teamId = premiumStatus.b2bStatus?.currentTeam?.teamId;
+    if (!teamId) {
+        throw new Error('User not part of a team');
+    }
+    const response = await getTeamMembersRequest({
+        secrets,
+        teamId,
+        page,
+    });
+
+    console.log(JSON.stringify(response));
```

Comment:
Kind of strange to not return just the response here and do the display in the command but it's shorter.

Reply:
that's how it's done in other commands, we can consider changing it everywhere tho'